daemon/daemon_main.go

```diff
@@ -453,6 +453,7 @@ func init() {
 
 	flags.Bool(option.KeepBPFTemplates, false, "Do not restore BPF template files from binary")
 	option.BindEnv(option.KeepBPFTemplates)
+	flags.MarkDeprecated(option.KeepBPFTemplates, "This option is no longer supported and will be removed in v1.9")
```

Comment:
Awesome, please also add a paragraph to the upgrade guide. We maintain a section with all options that are being deprecated and what action users have to take.

Reply:
Updated commit 035bc332e9dfff907f5347e3367a3b394e910099 ("option: mark --keep-bpf-templates as deprecated") to include a paragraph in the upgrade guide.